pkg/templates/charts/toggle/search-v2-operator/templates/search-v2-operator-controller-manager.yaml

```diff
@@ -74,7 +73,7 @@ spec:
         name: manager
         readinessProbe:
           httpGet:
-            path: /readyz
+            path: /readyz3
```

Comment:
Once the search automation enablement is disabled remember to back out any of the search changes made and then it should be good to go